content/Django tips and good practices - Bruno Michetti.md

```diff
@@ -0,0 +1,164 @@
+# Django: tips and good practices
+
+As every software developer, I like to create maintainable code, not just for me but also for my partners.  It feels good to know that the programs you use and/or mantain, have good code quality. It feels good to know that we try to construct something easy to change, to reuse, to update, to review, etc. 
+
+I learned to program in the **Python** languange using the **Django** web and **Django REST** frameworks a few months ago, and I would like to share some tips and good practices for the development of REST APIs using those technologies. When I face a programing problem, first I try to solve it, and then I try to find out if there is a better way to reach the solution, and then I like to write that down or store that information, to share it with others, to publish that somewhere to help in the future to other person. I hope this blog reaches that goal: to help people learning Django, or at least to be used as a good example for those who want to code and have something to base on.
+
+## Don't reinvent the wheel
+Django is a good framework to develop web applications using the Python language. Is fast, secure, scalable and well documented. If you want to develop REST APIs, you can combine it with the Django REST framework, and you can have just in seconds, a base project generated by these tools. You can take a look at the documentation in these links:
```

Comment:
too much ausage of: "you can" , "you have", etc. 
Go straight: Take a look at the documentation....